great_expectations/rule_based_profiler/data_assistant/volume_data_assistant.py

```diff
@@ -37,6 +44,52 @@ def __init__(
             data_context=data_context,
         )
 
+    def plot(self, data_assistant_result: DataAssistantResult):
+        """
+        VolumeDataAssistant-specific plots are defined with Altair and passed to "super().plot()" for display.
+        """
+        metric_name: str = self.metrics_parameter_builders_by_domain_type[
+            MetricDomainTypes.TABLE
+        ][0]["metric_name"]
+        metric_label = metric_name.replace(".", " ").replace("_", " ").title()
+        x_axis_label: str = "Batch"
+
+        # available data types: https://altair-viz.github.io/user_guide/encoding.html#encoding-data-types
+        x_axis_type: str = "nominal"
+        metric_type: str = "quantitative"
+
+        fully_qualified_parameter_name: str = (
+            self.metrics_parameter_builders_by_domain_type[MetricDomainTypes.TABLE][0][
```

Comment:
@NathanFarmer All metric results are report by `Domain` (now a hashable object), which contain the `domain_type` fields.  So, I am hoping that we can move this to the result object.  Thanks.

Reply:
✅